extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/OidcTenantConfig.java

```diff
@@ -86,6 +86,26 @@
      */
     Authentication authentication = new Authentication();
 
+    /**
+     * TLS configurations
+     */
+    @ConfigItem
+    TlsConfig tls;
```

Comment:
@fmhwong Please change it to `Tls tls = new Tls()` and rename the group to `Tls` so that we can have `quarkus.oidc.tls.trust-all-certificates`, etc, the `config` is assumed :-)